exercises/list-ops/src/ListOps.hs

```diff
@@ -13,25 +13,25 @@ import Prelude hiding
   ( length, reverse, map, filter, foldr, (++), concat )
 
 foldl' :: (b -> a -> b) -> b -> [a] -> b
-foldl' = error "You need to implement this function."
+foldl' f z xs = error "You need to implement this function."
 
 foldr :: (a -> b -> b) -> b -> [a] -> b
-foldr = error "You need to implement this function."
+foldr f x xs = error "You need to implement this function."
```

Comment:
Oh! Sorry.

I didn't explicitly said earlier, but all the *folds* usually use `z`. Also, for consistency with the previous `foldl'`, this could use it too.